src/FffGcodeWriter.cpp

```diff
@@ -1261,15 +1250,15 @@ bool FffGcodeWriter::processInsets(const SliceDataStorage& storage, LayerPlan& g
                 // nothing to do
                 return false;
             }
-            const unsigned bottom_layers = mesh.getSettingAsCount("bottom_layers");
-            if (layer_nr >= bottom_layers)
+            const int bottom_layers = mesh.getSettingAsCount("bottom_layers");
```

Comment:
So the value cannot be negative. This change was just because it is compared with unsigned integers and it gave compiler warnings.